dandi/cli/cmd_organize.py

```diff
@@ -59,6 +61,13 @@
     ),
 )
 @click.argument("paths", nargs=-1, type=click.Path(exists=True))
+@click.option(
+    "--number-of-jobs",
+    "number_of_jobs",
+    type=int,
+    default=None,
+    help="The number of jobs to use during organization.",
+)
```

Comment:
```suggestion
@click.option("-J", "--jobs", type=int, help="Number of jobs during organization")
```
for consistency with other commands? (this one from `move` "as is")